### ISSUE TITLE
Add up/down key control to dropdown

### ISSUE THREAD

**Issue by VortrexFTW** [MexUK/mexui#9]
To select a dropdown item using keyboard only. GTAC cursor doesn't work in GTA IV.

**Comment by VortrexFTW** [comment]
Quick update: GTAC now has a working cursor for IV so this feature isn't as urgent now.
